Week1/JS_excercises/5_round_number.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+let z = 7.25;
+console.log(z);
+
+let a = Math.round(z);
+console.log(a);
+//
+let b = a > z ? a : z;
```

Comment:
Aahaan. You could use `const` keyword here instead of `let` since we are not reassigning any of these variables, right?

Reply:
That's true, 
I'll keep it in mind.